test/strategy/liquity/LiquityStrategy.spec.ts

```diff
@@ -557,17 +557,60 @@ describe('LiquityStrategy', () => {
       ).to.be.revertedWith('StrategyNothingToReinvest');
     });
 
-    it('fails if the minimum principal protection is not guaranteed', async () => {
-      await depositToVault(parseUnits('100'));
-      await strategy.setMinPrincipalProtectionPct(12000);
+    it('protects pricipal by reverting when total underlying assets are greater than min protected principal', async () => {
```

Comment:
I'm confused by this sentence. I guess it is protecting the principal by reverting when the reinvested amount is not enough to cover the principal?

Reply:
you have a point there